editor.planx.uk/src/routes/views/draft.tsx

```diff
@@ -32,12 +32,14 @@ export const draftView = async (req: NaviRequest) => {
   const flowData = await fetchDraftFlattenedFlowData(flow.id);
 
   const state = useStore.getState();
-  state.setFlow({ id: flow.id, flow: flowData, flowSlug });
-  state.setFlowNameFromSlug(flowSlug);
+  state.setFlow({ id: flow.id, flow: flowData, flowSlug, flowName: flow.name });
+  state.setFlowName(flow.name);
```

Comment:
similar to above comment, do we actually need both of these calls, or can it be simplified to just the first `setFlow({ ... })` one since that includes `flowName` now?

Reply:
Yeah, so removing ``setFlowName`` in favour of just using ``setFlow`` and pulling the ``name`` type from there?